src/main/java/com/bric/awt/BristleStroke.java

```diff
@@ -85,7 +86,7 @@ public BristleStroke(float width, float thickness, long randomSeed) {
         this.width = width;
         this.thickness = thickness;
         this.grain = getGrain(width, thickness);
-        this.spacing = 0.5f + 0.5f * thickness;
+        this.spacing = HALF_UNIT + HALF_UNIT * thickness;
```

Comment:
This doesn't make the code more readable to me. Also note that I consider all packages outside the "pixelitor" package to be "library code" (which for one reason or another isn't on Maven Central), and I try to make as few changes to it as possible.